graylog2-server/src/main/java/org/graylog/security/authservice/AuthServiceBackendConfig.java

```diff
@@ -40,7 +41,7 @@ default void validate(ValidationResult result) {
     }
 
     @JsonIgnore
-    List<String> hostAllowList();
+    Optional<List<String>> hostAllowList();
```

Comment:
As mentioned [here](https://github.com/Graylog2/graylog2-server/pull/15305#issuecomment-1519746783), we should give this method a more specific name to communicate the use case. Let's also add javadoc to the method to describe its use.

How about something like `externalHTTPHosts()`?

Reply:
I like the name - refactored it.